File: ClientApp/ClientApp/Dto/IdeaDto.cs
namespace ClientApp.Dto
{
    public class IdeaDto
    {
        public int? Id { get; set; }
        public string? Keyword { get; set; }
        public string? Difficulty { get; set; }
        public string? Volume { get; set; }
        public DateTime? LastUpdated { get; set; }
        public int? KeywordSuggestionsId { get; set; }
    }
}

File: KeywordSuggestionsApi/src/main/java/hr/algebra/keywordsuggestionsapi/dto/IdeaDto.java
package hr.algebra.keywordsuggestionsapi.dto;

import jakarta.persistence.Column;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDateTime;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class IdeaDto {
    private Integer id;
    private String keyword;
    private String difficulty;
    private String volume;
    private LocalDateTime lastUpdated;

    private Integer keywordSuggestionsId;
}
